helpers/any.js

```diff
@@ -15,27 +14,10 @@ const factory = () => {
         const opts = args.pop();
         const predicate = opts.hash;
 
-        if (!_.isEmpty(predicate)) {
+        if (Object.keys(predicate).length > 0) {
             // With options hash, we check the contents of first argument
             any = _.any(args[0], predicate);
-        } else {
```

Comment:
Not sure we want to remove this. I don't know that we've announced this deprecation anywhere.

Reply:
Unremoved.